kalite/distributed/static/css/distributed/khan-lite.css

```diff
@@ -820,50 +795,24 @@ a.disabled {
 }
 
 #software-keyboard div {
```

Comment:
Could it be `#software-keyboard .row {`? Small change, I know, but more semantic.